packages/edit-site/src/components/sidebar-navigation-screen-pattern/template-part-navigation-menu-list.js

```diff
@@ -10,9 +10,9 @@ import TemplatePartNavigationMenuListItem from './template-part-navigation-menu-
 export default function TemplatePartNavigationMenuList( { menus } ) {
 	return (
 		<ItemGroup className="edit-site-sidebar-navigation-screen-template-part-navigation-menu-list">
-			{ menus.map( ( menuId ) => (
+			{ [ ...new Set( menus ) ].map( ( menuId, index ) => (
 				<TemplatePartNavigationMenuListItem
-					key={ menuId }
```

Comment:
Generally it would be far better to use a unique ID rather than an index as we are already doing. Moreover, [the `Array.map()` method already includes an `index` method on its callback](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map#parameters).

Reply:
Why is a unique id better than an index? Also not that what we are currently doing is not a unique ID as the same navigation can appear on the same template more than once.

Reply:
See: https://react.dev/learn/rendering-lists#why-does-react-need-keys